x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/datafeed_anomalous_error_rate_for_user_agents_jsbase.json

```diff
@@ -0,0 +1,27 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "max_empty_searches": 10,
+  "query": {
+    "bool": {
+      "must": [
```

Comment:
As above, try and condense these into single lines for each part of the `must` clause
```
"must": [
  { "bool": { "filter": { "term": { "agent.name": "js-base" } } } },
  { "exists": { "field": "user_agent.name" } }
]
```

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69